torch/_inductor/compile_fx.py

```diff
@@ -340,6 +340,7 @@ def compile_fx(model_: torch.fx.GraphModule, example_inputs_: List[torch.Tensor]
         model_ = normalize_ir(model_, example_inputs_)
         model_ = overrides.replace_fx(model_)
         model_ = overrides.fuse_fx(model_, example_inputs_)
+        model_ = overrides.fxFuser(model_, example_inputs_)
```

Comment:
This seems to be doing a lot of the same things as the fuse_fx line right above it (which is CPU-only) is doing.  There are also a few pending PRs there (see this PR stack: https://github.com/pytorch/pytorch/pull/87064)

Can we refactor `fuse_fx` and `fxFuser` to share more code?

cc @XiaobingSuper

Reply:
Yes, we could integrate fusion of `fxFuser` into `fuse_fx`. I will place the fusion of `fxFuser` before `if torch.is_grad_enabled():', because it supports both inference and training and not is limited to CPU.